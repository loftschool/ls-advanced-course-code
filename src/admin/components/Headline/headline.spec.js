import headline from "./Headline.vue";
import { shallowMount } from "@vue/test-utils";

it("сохраняет структуру", async () => {
  const component = shallowMount(headline);

  expect(component.element).toMatchSnapshot();
});

it("скрывает блок содержимого", () => {
  const component = shallowMount(headline, {
    propsData: {
      hideContent: true
    }
  });

  expect(component.element).toMatchSnapshot();
})
