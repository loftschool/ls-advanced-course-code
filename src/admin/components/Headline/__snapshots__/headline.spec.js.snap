// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`скрывает блок содержимого 1`] = `
<div
  class="headline-component"
>
  <!---->
  <button
    class="logout"
    type="button"
  >
    Выйти
  </button>
</div>
`;

exports[`сохраняет структуру 1`] = `
<div
  class="headline-component"
>
  <div
    class="content"
  >
    <div
      class="desc"
    >
      Панель администрирования
    </div>
  </div>
  <button
    class="logout"
    type="button"
  >
    Выйти
  </button>
</div>
`;
